assets/src/js/event-actions.js

```diff
@@ -240,7 +240,8 @@
 				$( '#events' ).replaceWith( response.data );
 				advanced_cron_manager.slidebar.form_process_stop();
 				advanced_cron_manager.slidebar.close();
-				wp.hooks.doAction( 'advanced-cron-manager.event.sort' );
+				wp.hooks.doAction( 'advanced-cron-manager.event.search.do-trigger' );
+				wp.hooks.doAction( 'advanced-cron-manager.event.sort.do-trigger' );
```

Comment:
It seems like an anti-pattern here ;) you are `doing an action` and label it as an `trigger` which is redundant.

What would you say to keep it simple?

```suggestion
				wp.hooks.doAction( 'advanced-cron-manager.event.search' );
				wp.hooks.doAction( 'advanced-cron-manager.event.sort' );
```